python/matrix/matrix.py

```diff
@@ -5,15 +5,10 @@ def __init__(self, matrix_string):
         for i in lines:
             b = [int(l) for l in i.split()]
             a.append(b)
-        self.matrix = a
+        self.matrix = a:
 
     def row(self, index):
         return self.matrix[index-1]
```

Comment:
Again, formatting before pushing is a good habit: whitespace is required around arithmetic operators.